src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Library Controls
     QMenu* libraryMenu = addSubmenu(tr("Library"));
-    addPrefixedControl("[Playlist]", "ToggleSelectedSidebarItem",
-                       tr("Expand/Collapse View"),
-                       tr("Expand/collapse the selected view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveUp",
+                       tr("Move up"),
+                       tr("Equivalent to pressing the UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-
-    addPrefixedControl("[Playlist]", "SelectPlaylist",
-                       tr("Switch Next/Previous View"),
-                       tr("Switch to the next or previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveDown",
+                       tr("Move down"),
+                       tr("Equivalent to pressing the DOWN key on the keyboard"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "MoveVertical",
+                       tr("Move up/down"),
+                       tr("Move vertically in either direction using a knob, as if pressing UP/DOWN keys"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
```

Comment:
> As we previously discussed, my understanding is that currently UP/DOWN maps to slotSelectTrack(), and LEFT/RIGHT maps to slotSelectSidebarItem(). Furthermore, there are two buttons LoadA and LoadB that load the currently selected track into Deck A / Deck B respectively. Is this correct?

Yes. The current mapping is a pain. 

But I do not think that a solution that mandatory requires shift is even better. In this case I will stick with the current mapping, Since it is easier to control. 

In master: 
* Right/Left to select the playlist
* Up/Down to select the track 
* LoadA/LoadB to load the track 

Your proposal: 
* Up/Down to select the feature
* [Shift + Right] to open a tree child 
* Up/Down to select a playlist. 
* Right to switch focus to the track table 
* Up/Down to select the track 
* LoadA/LoadB to load the track 

I hope you see that there might be users who want to be able to map th old controls.
It is a bit annoying to step over the expanded tree though. 

The proposal for the new library layout may look like this: 
* Up/Down to select the feature
* Right to switch to the feature pane 
* Up/Down to select a playlist. 
* Right to switch focus to the track table 
* Up/Down to select the track 
* LoadA/LoadB to load the track

No Shift would be required. :-D